modules/exploits/linux/http/cisco_rv130_rmi_rce.rb

```diff
@@ -122,6 +281,85 @@ def send_request(buffer)
     end
   end
 
+  def check
+
+    # We fingerprint devices using SHA1 hash of a web resource accessible to unauthenticated users.
+    # We use lang_pack/EN.js because it's the one file that changes the most between versions.
+    # Note that it's not a smoking gun given that some branches keep the exact same files in /www
+    # (see RV110 branch 1.2.1.x/1.2.2.x, RV130 > 1.0.3.22, RV215 1.2.0.x/1.3.x)
+
+    fingerprints = {
+      "69d906ddd59eb6755a7b9c4f46ea11cdaa47c706" => {
+        "version" => "Cisco RV110W 1.1.0.9",
+        "status" =>Exploit::CheckCode::Vulnerable
+      },
+      "8d3b677d870425198f7fae94d6cfe262551aa8bd" => {
+        "version" => "Cisco RV110W 1.2.0.9",
+        "status" => Exploit::CheckCode::Vulnerable
+      },
+      "134ee643ec877641030211193a43cc5e93c96a06" => {
+        "version" => "Cisco RV110W 1.2.0.10",
+        "status" => Exploit::CheckCode::Vulnerable
+      },
+      "e3b2ec9d099a3e3468f8437e5247723643ff830e" => {
+        "version" => "Cisco RV110W 1.2.1.4, 1.2.1.7, 1.2.2.1 (not vulnerable), 1.2.2.4 (not vulnerable)",
+        "status" => Exploit::CheckCode::Unknown
+      },
+      "6b7b1e8097e8dda26db27a09b8176b9c32b349b3" => {
+        "version" => "Cisco RV130/RV130W 1.0.0.21",
+        "status" => Exploit::CheckCode::Vulnerable
+      },
+      "9b1a87b752d11c5ba97dd80d6bae415532615266" => {
+        "version" => "Cisco RV130/RV130W 1.0.1.3",
+        "status" => Exploit::CheckCode::Vulnerable
+      },
+      "9b6399842ef69cf94409b65c4c61017c862b9d09" => {
+        "version" => "Cisco RV130/RV130W 1.0.2.7",
+        "status" => Exploit::CheckCode::Vulnerable
+      },
+      "8680ec6df4f8937acd3505a4dd36d40cb02c2bd6" => {
+        "version" => "Cisco RV130/RV130W 1.0.3.14, 1.0.3.16",
+        "status" => Exploit::CheckCode::Vulnerable
+      },
+      "8c8e05de96810a02344d96588c09b21c491ede2d" => {
+        "version" => "Cisco RV130/RV130W 1.0.3.22, 1.0.3.28, 1.0.3.44, 1.0.3.45 (not vulnerable), 1.0.3.51 (not vulnerable)",
+        "status" => Exploit::CheckCode::Unknown
+      },
+      "2f29a0dfa78063d643eb17388e27d3f804ff6765" => {
+        "version" => "Cisco RV215W 1.1.0.5",
+        "status" => Exploit::CheckCode::Vulnerable
+      },
+      "e5cc84d7c9c2d840af85d5f25cee33baffe3ca6f" => {
+        "version" => "Cisco RV215W 1.1.0.6",
+        "status" => Exploit::CheckCode::Vulnerable
+      },
+      "7cc8fcce5949a68c31641c38255e7f6ed31ff4db" => {
+        "version" => "Cisco RV215W 1.2.0.14 or 1.2.0.15",
+        "status" => Exploit::CheckCode::Vulnerable
+      },
+      "050d47ea944eaeadaec08945741e8e380f796741" => {
+        "version" => "Cisco RV215W 1.3.0.7 or 1.3.0.8, 1.3.1.1 (not vulnerable), 1.3.1.4 (not vulnerable)",
+        "status" => Exploit::CheckCode::Unknown
+      }
+    }
+
+    uri = target_uri.path
+    res = send_request_cgi({
+      'method' => 'GET',
+      'uri' => normalize_uri(uri, 'lang_pack/EN.js')
+    })
+    if res and res.code == 200
```

Comment:
`&&` is preferred over `and`.

https://github.com/rubocop-hq/ruby-style-guide#no-and-or-or

Reply:
See https://github.com/rapid7/metasploit-framework/pull/12133/commits/610bed8fd9a6d0cc6d0174c60520a421ecf01403